src/manageiq_client/api.py

```diff
@@ -363,13 +384,14 @@ def reload(self, expand=None, get=True, attributes=None):
             if isinstance(attributes, six.string_types):
                 attributes = [attributes]
             kwargs.update(attributes=",".join(attributes))
-        if get:
+        if "href" in self._data:
+            self._href = self._data["href"]
+        if get and self._href:
```

Comment:
If we raise error, we would make it impossible to work with resources in e.g. /api/automate. We can't derive href from id, /api/automate/:id is invalid.
It could be possible to check if `get=True` is explicit or if it's just the default value, but I'm not sure it's necessary.